test/cppsuite/test_harness/workload/thread_context.h

```diff
@@ -71,16 +71,19 @@ class transaction_context {
     void begin(const std::string &config = "");
     /* Begin a transaction if we are not currently in one. */
     void try_begin(const std::string &config = "");
-    void commit(const std::string &config = "");
-    /* Attempt to commit the transaction given the requirements are met. */
-    void try_commit(const std::string &config = "");
+    /*
+     * Commit a transaction, returns a boolean indicating if rollback is required.
```

Comment:
Can we reword to:
`Commit a transaction and return a boolean if rollback is required.`